srcpkgs/linux4.19/files/ppc64-dotconfig

```diff
@@ -70,17 +67,17 @@ CONFIG_NO_HZ_IDLE=y
 # CONFIG_NO_HZ is not set
 CONFIG_HIGH_RES_TIMERS=y
 # CONFIG_PREEMPT_NONE is not set
-CONFIG_PREEMPT_VOLUNTARY=y
-# CONFIG_PREEMPT is not set
+# CONFIG_PREEMPT_VOLUNTARY is not set
+CONFIG_PREEMPT=y
```

Comment:
Preempt kernel is a no at least for openpower hardware (power8+), please use `PREEMPT_VOLUNTARY` like before